.vscode/launch.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
Let's not keep this files in the repo. The same for `.vscode/tasks.json`.

Reply:
We should add `.vscode` to `.gitignore`